modules/common/src/main/scala/surge/health/config/WindowingStreamConfig.scala

```diff
@@ -51,9 +54,18 @@ object WindowingStreamConfigLoader {
     val maxStreamSize = config.getInt("surge.health.window.stream.max-size")
     val frequencies = config.getDurationList("surge.health.window.stream.frequencies").asScala.map(d => FiniteDuration(d.toMillis, "millis"))
 
+    val drainPerTick = Try { config.getInt("surge.health.stream.drain-per-tick") }.toOption.getOrElse(10)
```

Comment:
No need to wrap these in a `Try` - we should always have default values defined for these so it shouldn't fail. If someone horribly misconfigures it also may be better to explode than quietly ignore their setting & use a default

Reply:
done